app/src/main/java/com/hedvig/app/feature/keygear/ui/ReceiptActivity.kt

```diff
@@ -0,0 +1,37 @@
+package com.hedvig.app.feature.keygear.ui
+
+import android.content.Intent
+import android.os.Bundle
+import com.bumptech.glide.Glide
+import com.hedvig.app.BaseActivity
+import com.hedvig.app.R
+import kotlinx.android.synthetic.main.activity_receipt.*
+
+class ReceiptActivity : BaseActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_receipt)
+
+        val url = "https://upload.wikimedia.org/wikipedia/commons/0/0b/ReceiptSwiss.jpg"
```

Comment:
Let's make this a parameter passed to the Activity so that we can plug in the 'real' receipts when we have them 🙂